week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,8 @@
 // Declare your function here
+function createLongGreeting(){
+    return "my name is";
 
-const greeting = createLongGreeting("Daniel", 30);
+
+const greeting = createLongGreeting("Daniel", "i am " + "30");}
```

Comment:
Can you double check if this code runs? I think it doesn't - can you tell me why?